### ISSUE TITLE
Valid numbers shown invalid - Finland

### ISSUE THREAD

**Issue by swapnilk1** [googlei18n/libphonenumber#1762]
Hi 

Numbers +3585062301 and +358502218 marked invalid by libphonenumber-8.4.3 version in Java version. I can see that both numbers are valid mobile numbers. I used isValidNumber() method to test this.

Thanks-Swapnil

**Comment by kkeshava** [comment]
@jonbom,
Thank you for taking the time to improve libphonenumber!
For your next metadata issue, we invite you to file issues in Google's new Issue Tracker, which we're currently testing.

We can't connect to +3585062301 and +358502218.

We support 8 to 10 digit long numbers for the prefix 50 while the specified number is 6 and 7 digits long.

Here is our official evidence: https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000490002PDFE.pdf

We are not able to find 6 and 7 digit numbers with this prefix online.

Please share any working numbers or official evidence you may have. See our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).

**Comment by kkeshava** [comment]
This issue is blocked on your or other GitHub contributors' input. Please help us find evidence that 50 (6 and 7 digits long) is valid, or provide an example number from a business website or similar. Otherwise we don't have enough information to fix the issue and may have to close it as infeasible.

**Comment by jonbom** [comment]
@kkeshava
I think you mixed me up with @swapnilk1 :)

**Comment by kkeshava** [comment]
Oh so sorry :)

**Comment by keghani** [comment]
We did find evidence and fixed this in 8.5.2.
